persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -96,17 +109,27 @@ public void execute() {
             errStream.println(commandUsageInfo);
             return;
         }
+        Path persistTomlPath = Paths.get(this.sourcePath, SUBMODULE_PERSIST, PERSIST_TOML_FILE);
+        Path databaseConfigurationBalPath = Paths.get(this.sourcePath, SUBMODULE_MODULES, KEYWORD_CLIENTS,
+                DATABASE_CONFIGURATION_BAL);
+        File persistToml = new File(persistTomlPath.toString());
+        File databaseConfigurationsBal = new File(databaseConfigurationBalPath.toString());
+        if (!persistToml.exists() || !databaseConfigurationsBal.exists()) {
+            errStream.println("Persist project is not initiated. Please run `bal persist init` " +
+                    "to initiate the project before the database schema generation");
+            return;
+        }
 
         try  {
             balProject = ProjectLoader.loadProject(Paths.get(sourcePath));
             name = balProject.currentPackage().descriptor().org().value() + "." + balProject.currentPackage()
                     .descriptor().name().value() + "." + "clients";
-            setupJdbcDriver();
 
             balProject = BuildProject.load(Paths.get(sourcePath).toAbsolutePath());
             balProject.currentPackage().getCompilation();
             configurations = SyntaxTreeGenerator.readToml(Paths.get(this.sourcePath, this.configPath), name);
             sqlLines = readSqlFile();
+            setupJdbcDriver();
```

Comment:
```suggestion
            loadJdbcDriver();
```

Reply:
Updated in [5917ac9](https://github.com/ballerina-platform/persist-tools/pull/53/commits/5917ac90bacafc1e6950859e4fbf887010683c60)